New principle (#2)
trustpilot/principles: README.md

Always apply open source best practices to all repositories, public and private ([Inner Source](https://en.wikipedia.org/wiki/Inner_source)). That typically includes a clear documentation (README) and code that isn’t tightly coupled to anything internally. Consider which pieces could be pulled out and open sourced (e.g. a general purpose library without any dependencies is a prime candidate to be open source).

*We do this because code written with open source on mind, is usually cleaner, better documented and not tightly coupled to anything internally. Generally it’s a good sanity check that we’re not doing anything too weird internally. Additionally, open source is a great recruiting tool for us to display our engineering culture.*

> ## Own Your Products

Reviewer: I'm not sure the title and the description completely match. The description is about monitoring, observing, quality, state of code whereas the title is much more broad. Too broad IMO. Other (working) titles:

- Know the state of your products
- Keep your products healthy
- Own your products' health
- Your products, your responsibility
- ...

Reviewer 2: I agree. Which would be your best suggestion for the title?

Reviewer 3: How about the well know _You build it, you run it_

Reviewer: Of my own suggestions, the latter: "Your products, your responsibility". I'm fine with Allans suggestion as well and I'm sure others would be fine as well.

Reviewer 4: I like the existing one. It might be broad, but that is also the point of it. It relates to the Full Ownership concept, and while "products" might be a bit broad, it should also be broad, if we believe in the word "full".

> Be responsible for your products, internal and external, their artifacts and their service quality. Have good observability of your products, know the state of your services and when they need attention.

Reviewer: ```suggestion
Take responsibility for your products, internal and external, their artifacts and their service quality. Ensure good observability of your products, know the state of your services and when they need attention.
```

I think we could write this in a more active voice. Rather than the passive "Be responsible", I think "Take responsibility" better says that it's something we actively seek to do, and that responsibility for our products don't just land in our laps. I also like that it has a counter point - in another organisation they might **not** want to take responsibility but instead have centralised monitoring and disaster recovery. Something we're trying to have less of.

> Be responsible for your products, internal and external, their artifacts and their service quality. Have good observability of your products, know that state of your services and when they need attention.

Reviewer 2: ```suggestion
Be responsible for your products, internal and external, their artifacts and their service quality. Have good observability of your products, know the state of your services and when they need attention.
```
Typo 🙈

> Take responsibility for your products, internal and external, their artifacts and their service quality. Ensure good observability of your products, know the state of your services and when they need attention.

Reviewer: ```suggestion
Take responsibility for your products, internal and external, their artifacts and their service quality. Ensure good observability of your products, know the state of your services and when they need attention. You are responsible for keeping the products healthy and for knowing what that takes.
```

Below we mention testing, documentation and infrastructure but it's not really covered above. This suggestion tries to clear that out by adding that as a team to know when unit tests (fx) is very necessary and when it's not etc.

> We monitor our services, we define service level indicators, objectives and setup alerting if objectives is violated. We react promptly to these alerts when they happen, to improve our service quality or adjust the alerting threshold to reflect reality. Through monitoring and alerting we strive to maintain a high service quality.

Reviewer 5: Could this be reformulated in a more generic way? I don't think this principle would affect my everyday work in any way. Maybe we can say something more in the lines of:

```
We collect data of how our software performs, analyse it, and act on it. We do this at the cost of speed, but we never make decisions on unfounded assumptions.
```

Principles aren't useful if they tell us something that everyone would agree is good. They should exist for situations when you're deciding between 2 good things, but we as an organization discovered that one of them works best for us.
